test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/IncompleteDirectives_DesignTime.mappings.txt

```diff
@@ -20,21 +20,21 @@ Generated Location: (830:27,0 [0] )
 
 Source Location: (159:11,8 [17] TestFiles/IntegrationTests/CodeGenerationIntegrationTest/IncompleteDirectives.cshtml)
 |MyService<TModel>|
-Generated Location: (883:30,0 [17] )
+Generated Location: (980:31,0 [17] )
```

Comment:
generated locations changing because the generated code includes additional pragmas.